supervisor/api/addons.py

```diff
@@ -347,21 +348,23 @@ async def options_validate(self, request: web.Request) -> None:
             data[ATTR_MESSAGE] = humanize_error(addon.options, ex)
             data[ATTR_VALID] = False
 
-        # Validate security
-        if self.sys_config.force_security:
-            for secret in addon.pwned:
-                try:
-                    await check_pwned_password(self.sys_websession, secret)
-                    continue
-                except PwnedSecret:
-                    data[ATTR_MESSAGE] = "Add-on use pwned secrets!"
-                except PwnedError as err:
-                    data[
-                        ATTR_MESSAGE
-                    ] = f"Error happening on pwned secrets check: {err!s}!"
-
-                data[ATTR_VALID] = False
-                break
+        # Pwned check
```

Comment:
Shouldn't we check if the `addon_pwned` check is enabled before doing this check?

Reply:
We should not use the resolution center value to enable or disable features. I think the issue was the persistent message, this will be still only showing a small warning but don't block actions. We can add a pwned flag to supervisor options if we want to skeep this.